### DISCUSSION TITLE
Mastodon posts shortened URLs

### DISCUSSION THREAD

**Discussion by sudhagar-mr-17051** [mastodon/mastodon#31493] [General]
Hi team,

While going through the documentation, we found some instructions on how links are managed in posts (https://docs.joinmastodon.org/user/posting/#links). When publishing a post with a shortened URL, the post is being published successfully. However, the documentation states that shortened URLs are discouraged. We would like to know in what context shortened URLs are discouraged. Will post reach be reduced? Is there any probability of post failure?

Thanks in advance.

**Comment by quicoto** [comment]
I think that's meant on a privacy/tracking topic. Like short users are usually used when you want to track clicks, which invades privacy for which the Fediverse is very keen on.

As far as I understand there is no "reach" (not algorithm) on the Fediverse, so your post will be federated all the same independent of the URL.

Not sure if this helps

**Reply by sudhagar-mr-17051** [reply]
Thank you so much for taking the time to explain this! Your insights were incredibly helpful, and I really appreciate the clarity you brought to the topic.
